### PR TITLE
Serving Invenio specialized vocabularies (like names) via the standard /api/vocabularies endpoint

### PR DESCRIPTION
**Pull request by mesemus**
Added cfg option 

```
    OAREPO_VOCABULARIES_SPECIALIZED_SERVICES = {
        "names": "names",
    }
```

which creates "/api/vocabularies/names" as a duplicate endpoint for /api/names. Apart from "search in all vocabularies" this endpoint provides all the methods "normal" vocabularies provide.

### CODE REVIEW

File: oarepo_vocabularies/fixtures.py [hunk 1]
```diff
@@ -67,4 +67,7 @@ def write(self, stream_batch: StreamBatch, *args, **kwargs):
         return super().write(stream_batch, *args, **kwargs)
 
     def _get_stream_entry_id(self, entry):
-        return (self.vocabulary, super()._get_stream_entry_id(entry))
+        _id = super()._get_stream_entry_id(entry)
+        if _id:
+            return (self.vocabulary, super()._get_stream_entry_id(entry))
```
Comment (mirekys) [inline oarepo_vocabularies/fixtures.py:72]: Shouldn't be just `(self.vocabulary, _id)`?

File: oarepo_vocabularies/services/service.py [hunk 2]
```diff
@@ -78,6 +81,71 @@ def _vocabulary_statistics(self):
...
+        specialized_service = self.get_specialized_service(type)
+        if specialized_service:
+            return specialized_service.read_many(identity=identity, ids=ids, fields=fields, **kwargs)
+        return super().read_many(identity=identity, type=type, ids=ids, fields=fields, **kwargs)
+
+    @unit_of_work()
+    def create(self, identity, data, uow=None, expand=False, **kwargs):
+        specialized_service = self.get_specialized_service(data.get("type"))
+        if specialized_service:
+            data.pop("type")
```
Comment (mirekys) [inline oarepo_vocabularies/services/service.py:130]: Shouldn't we pop every key not matching target service schema here (like `props`, `title`)?

Comment (mesemus) [reply oarepo_vocabularies/services/service.py:130]: no, the "type" in the payload is there just to be able to select the vocabulary, the rest of the items must match the vocabulary schema.
That's why only the type is popped, the rest must be according to the schema.

**Review by mirekys — CHANGES_REQUESTED** [review]
Please fix sonar issue :)

**Review by mirekys — APPROVED** [review]

**Pull request merged**